python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -636,6 +653,10 @@ def get_nodes_for(
 ) -> (Dict[NodeType, int], List[ResourceDict]):
     """Determine nodes to add given resource demands and constraints.
 
+    In the event that multiple nodes would result in the same utilization, ties
+    are guaranteed to be broken by picking the first element from `node_types`
+    (note that python dicts are ordered).
```

Comment:
Ehm...according to the internet, this is an official language feature only as of Python 3.7.
If there's a convenient way of avoiding using that, that'd be good. If not, then whatever yolo.

Reply:
updated to use `collections.OrderedDict`. Can't wait to drop python 3.6